pkg/testers/node/node.go

```diff
@@ -61,6 +60,8 @@ type Tester struct {
 	ImageConfigFile                string `desc:"Path to a file containing image configuration."`
 	ImageConfigDir                 string `desc:"Path to image config files."`
 	Parallelism                    int    `desc:"The number of nodes to run in parallel."`
+	GcpProjectType                 string `desc:"Explicitly indicate which project type to select from boskos."`
```

Comment:
nit: use `GCPProjectType` instead of `GcpProjectType`

Reply:
👍 output would be same for both, but `GCPProjectType` seems nicer.